### PR TITLE
adding help text and index type to data dictionary field

### PR DESCRIPTION
**Pull request by cam156**
refs #326

### CODE REVIEW

File: app/cho/data_dictionary/with_index_type.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module DataDictionary::WithIndexType
+  extend ActiveSupport::Concern
+
+  IndexTypes = Valkyrie::Types::String.enum('facet', 'none')
```
Comment (awead) [inline app/cho/data_dictionary/with_index_type.rb]: Is "none" in this case not faceted, or not indexed at all?

Comment (cam156) [reply app/cho/data_dictionary/with_index_type.rb]: @awead not faceted.  I can change it to that.

Comment (awead) [reply app/cho/data_dictionary/with_index_type.rb]: @cam156 thanks, that would clear it up!

**Comment by cam156** [comment]
@awead it has been updated

**Comment by cam156** [comment]
@mtribone I figure this is not a final UI, so making it pretty is not really in scope with adding fields.  I'm not even sure this screen will live in the final product.  Did you want to add a ticket and mark all the things wrong with this screen or @awead  can create a ticket to wireframe the screen if we decide it will stay?

**Comment by awead** [comment]
@mtribone I've put in a ticket and slotted it for MVP 1, but we can move it around later if needed.

**Pull request merged**